scipy/special/_basic.py

```diff
@@ -2953,6 +2956,92 @@ def factorialk(n, k, exact=True):
         raise NotImplementedError
 
 
+def stirling2(N, K, exact=True):
+    """Generate Stirling number(s) of the second kind.
+
+        Stirling numbers of the second kind count the number
+        of non-empty subsets that can be formed by N elements
+        into K non-emptuy subsets.
+
+        This is often expressed as "N subset K".
+
+        Parameters
+        ----------
+        N : int, ndarray
+            Number of things.
+        K : int, ndarray
+            Number of non-empty subsets taken.
+        exact : bool, optional
+            For integers, if `exact` is False, then floating point precision is
+            used, otherwise the result is computed exactly. For non-integers, if
+            `exact` is True, is truncated. Defaults to True.
+
+        Returns
+        -------
+        val : int, float, ndarray
+            The total number of combinations.
+
+        See Also
+        --------
+        binom : Binomial coefficient considered as a function of two real
+                variables.
```

Comment:
binomial != stirling2

Reply:
> binomial != stirling2

This is a `See Also` section of the docs, it provides ***related*** methods the user might be interested in using or reading about. Unless I'm misunderstanding the comment?

Reply:
Woops, I was confusing the `See Also` with the `Returns` section 😅.

Nonwithstanding,  "the total number of combinations" actually does sound like the description of the binomial coefficient. I believe that something like "partitions" would be more accurate description than "combinations".

Reply:
> Woops, I was confusing the `See Also` with the `Returns` section 😅.
> 
> Nonwithstanding, "the total number of combinations" actually does sound like the description of the binomial coefficient. I believe that something like "partitions" would be more accurate description than "combinations".

Got it. Yeah the `"the total number of combinations"` is copy-pasta from binomial. I agree w/`partitions` will switch to that one-thanks.